src/LinuxTracing/SwitchesStatesNamesVisitorTest.cpp

```diff
@@ -389,34 +396,57 @@ TEST_F(SwitchesStatesNamesVisitorTest,
 }
 
 void SwitchesStatesNamesVisitorTest::ProcessFakeEventsForThreadStateTests() {
-  // kTid1
+  // kPid1, kTid1
```

Comment:
This comment should be moved down right before `PerfEvent{MakeFakeSchedWakeupPerfEvent(kTid1, kWakeTimestampNs1)}.Accept(&visitor_);`, line 405.

Reply:
done.